### DISCUSSION TITLE
Custom Report - Parent Issue Summary

### DISCUSSION THREAD

**Discussion by smi60** [shridhar-tl/jira-assistant#321] [Q&A]
Hi @shridhar-tl,

I was wondering if it is possible to show Parent Issue Summary and Epic name (where issuetype is story or task) instead of Epic link in Custom Report (or any other reports).

I've been playing around with Expressions, but I was not able to get this info.

I've been using v2.54 Edge extension, and I've tried this on web app as well.

Use case: several tasks have identical summary and are places somewhere within project hierarchy (either as a story/task with Epic as a parent issue or as a subtask). All tasks are assigned to the same user and one has problems to identify the right task for logging work. Therefore parent issue and epic name would be helpful.

Thank you for your support and great work.

**Comment by shridhar-tl** [comment]
Hi @smi60 - It is possible in Worklog report, however I just realized that it is not possible in Custom Report as of now. I will see if I could add that option by this week end.

**Reply by smi60** [reply]
Thank you @shridhar-tl.
